src/Paysera/WalletApi/Mapper.php

```diff
@@ -1225,12 +1225,44 @@ protected function decodeAccount($data)
             $this->setProperty($account, 'userId', $data['user_id']);
         }
 
+        if (isset($data['ibans'])) {
+            $this->setProperty($account, 'ibans', $this->decodeIbans($data['ibans']));
```

Comment:
Technically `ibans` will always be an array of strings. Also, `trim` is really not needed - if there would be a white space, you should register a bug at Paysera Support.